sshuttle/client.py

```diff
@@ -907,6 +929,7 @@ def reloadAllAcls(self):
         AclHandler(self.redisClient, ACL_SOURCES_TYPE).reload_acl_file()
         AclHandler(self.redisClient, ACL_EXCLUDED_SOURCES_TYPE).reload_acl_file()
         AclHandler(self.redisClient, ALWAYS_CONNECTED_TYPE).reload_acl_file()
+        AclHandler(self.redisClient, ACL_ALWAYS_CONNECTED_TYPE).reload_acl_file()
```

Comment:
It is a little bit confusing here....we have "AclHandler(self.redisClient, ALWAYS_CONNECTED_TYPE).reload_acl_file()" call, and then, we have ACL_ALWAYS_CONNECTED_TYPE....

Reply:
Maybe a comment would help? Because this does reflect the requirements. The Admin must turn on the room feature Always Connected and the workspace must also be in the ACL - the desktop Always Connected flag must be set.

Reply:
I agree...or maybe rename the variable would help....

Reply:
I think the names are accurate, so I added comments to help with the usage.